store/storage/sqlite/db.go

```diff
@@ -19,6 +20,9 @@ const (
 	reservedStoreKey = "_RESERVED_"
 	keyLatestHeight  = "latest_height"
 
+	// TODO: it is a random number, need to be tuned
+	defaultBatchBufferSize = 100000
```

Comment:
For the best snapshot import performance we should consider tuning the WAL truncate to something big, on the order of single digit gigabytes.

Reply:
also `synchronous=off` will boost throughput: https://www.sqlite.org/pragma.html#pragma_synchronous